d2l-dom.html

```diff
@@ -31,7 +31,7 @@
 				if (node.shadowRoot) {
 					node = node.shadowRoot;
 				}
-				nodes = node.children;
+				nodes = node.children || {};
```

Comment:
Note: `children` was originally chosen since it returns elements, whereas `childNodes` returns nodes (ex. including whitespace, comments).  Oddly, line `38` further ensures non-elements are excluded.